src/routes/(public)/(guest)/education/+layout.svelte

```diff
@@ -0,0 +1,129 @@
+<script lang="ts">
+    import { app } from '$lib/stores/app';
+    import {
+        ArtworkDark,
+        ArtworkDarkMobile,
+        ArtworkLightMobile,
+        ArtworkLight
+    } from '$lib/images/github-education-program';
+
+    let artworkImageSrc;
+    getArtworkImageSrc();
+    monitorResize();
+
+    function getArtworkImageSrc() {
+        let isMobile = window.matchMedia('(max-width: 767px)').matches;
+        if ($app.themeInUse === 'light' && isMobile) {
+            artworkImageSrc = ArtworkLightMobile;
+        } else if ($app.themeInUse === 'light' && !isMobile) {
+            artworkImageSrc = ArtworkLight;
+        } else if ($app.themeInUse === 'dark' && isMobile) {
+            artworkImageSrc = ArtworkDarkMobile;
+        } else {
+            artworkImageSrc = ArtworkDark;
+        }
+    }
```

Comment:
could we use css classes for that?

we have util classes to hide on mobile or desktop as with dark/light mode 👍🏻

Reply:
Good one. Updated!